binrw/src/io/mod.rs

```diff
@@ -4,11 +4,15 @@
 //! used by binrw. In no_std environments, a compatible subset API is exposed
 //! instead.
 
+#[cfg(feature = "std")]
```

Comment:
Is there a reason our BufReader implementation is not available for no_std users?